scalatest/src/main/scala/org/scalatest/Engine.scala

```diff
@@ -857,7 +857,7 @@ private[scalatest] class PathEngine(concurrentBundleModMessageFun: => String, si
   // Used in each instance to track the paths of things encountered, so can figure out
   // the next path. Each instance must use their own copies of currentPath and usedPathSet.
   def getNextPath() = {
-    var next: List[Int] = null
+    var next: List[Int] | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.